src/commands/CmdCustom.cpp

```diff
@@ -272,6 +273,16 @@ int CmdCustom::execute (std::string& output)
     }
   }
 
+  std::string location  = (Context::getContext ().data_dir);
+  File pending_data = File (location + "/pending.data");
+  if (pending_data.exists()) {
+    Color warning = Color (Context::getContext ().config.get ("color.warning"));
+    std::cerr << warning.colorize (
+      format ("Found existing '*.data' files in {1}", location)) << "\n";
+    std::cerr << "  Taskwarrior's storage format changed in 3.0, requiring a manual migration.\n";
+    std::cerr << "  See https://github.com/GothenburgBitFactory/taskwarrior/releases.\n";
+    std::cerr << "  Remove the '*.data' files to silence this warning.\n";
```

Comment:
```suggestion
```
I still don't like this line for the same reasons as before: https://github.com/GothenburgBitFactory/taskwarrior/pull/3321#discussion_r1545527051

Reply:
Oh, I thought I removed that sentence at your request. Oops!